runtime/frequency/src/lib.rs

```diff
@@ -1283,8 +1283,7 @@ mod benches {
 		[pallet_time_release, TimeRelease]
 		[pallet_capacity, Capacity]
 		[pallet_frequency_tx_payment, FrequencyTxPayment]
-		// Todo: uncomment after removing the feature flag
-		// [pallet_passkey, Passkey]
+		[pallet_passkey, Passkey]
```

Comment:
Do we also need update the `pallets/passkey/src/weights.rs` file for this as well (beyond what happens at release time?)

Reply:
Yes, I'm planning to run it right now.